pkg/metrics/ensuarance.go

```diff
@@ -63,7 +63,8 @@ const (
 )
 
 var (
-	/**** Metrics related to crane-agent execution ****/
+
+	// LastActivity used to record the last activity time of each steps
```

Comment:
`// LastActivity records the last activity time of each steps`
remove `used to`

Reply:
done~